libnetwork/datastore/datastore.go

```diff
@@ -114,7 +114,7 @@ type ScopeClientCfg struct {
 const (
 	// LocalScope indicates to store the KV object in local datastore such as boltdb
 	LocalScope = "local"
-	// GlobalScope indicates to store the KV object in global datastore such as etcd
+	// GlobalScope indicates to store the KV object in global datastore
 	GlobalScope = "global"
```

Comment:
Possibly, this `const` (and its use) can also be removed, but leaving that for a separate PR (as it's used in quite some places)